lib/src/components/NxTag/types.ts

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import {ReactNode, WeakValidationMap, HTMLAttributes} from 'react';
+import * as PropTypes from 'prop-types';
+
+export const NX_TAG_COLORS =
+    ['light-blue', 'purple', 'pink', 'blue', 'red', 'green', 'orange', 'yellow', 'lime', 'indigo'] as const;
+export type NX_TAG_COLORS_TYPE = (typeof NX_TAG_COLORS)[number]; // See https://stackoverflow.com/a/45486495
+
+export interface Props extends Omit<HTMLAttributes<HTMLDivElement>, 'onSelect'> {
```

Comment:
We need to document that Div attributes are accepted

Reply:
> We need to document that Div attributes are accepted

https://github.com/sonatype/sonatype-react-shared-components/pull/290/commits/bc000299bcbf137fe9321b230dbc7cadb6d6a33d